deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/build.ps1

```diff
@@ -0,0 +1,20 @@
+$args = @(
+    "Product.wxs",
+    "WixUI_InstallDir_Custom.wxs",
+    "-out", "output\",
+    "-dPlatform=x64",
+    "-dProductVersion=0.2.0",
+    "-dConfiguration=Release",
+    "-ext", "WixUIExtension"
+)
+
+Start-Process -NoNewWindow -Wait -FilePath "${Env:ProgramFiles(x86)}\WiX Toolset v3.14\bin\candle.exe" -ArgumentList $args
```

Comment:
It looks the the WiX installer sets the `WIX` environment variable, which in my machine is currently `C:\Program Files (x86)\WiX Toolset v3.11\`. Should we use this variable instead?